weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -230,7 +267,13 @@ def get_splitter(file_path: str, output_info: OutFileInfo, dry_run: bool, force_
 
     if b'GRIB' in header:
         metrics.Metrics.counter('get_splitter', 'grib').inc()
-        return GribSplitter(file_path, output_info, force_split)
+
+        if '1' in use_version:
+            return GribSplitter(file_path, output_info, force_split)
+        elif '2' in use_version:
+            return GribSplitterV2(file_path, output_info, force_split)
```

Comment:
Have you checked how much faster v2 ends up being? If it is always faster, instead of adding the --version flag, should we instead check here if grib_copy can be found and if so always prefer that? I.e. remove the `--use-version` flag, and replace the '2' check here with with the `shutil.which(grib_copy)` check.

Reply:
I like this idea! Great shout.

Reply:
It may be faster, but the more important issue is that it should provide a more robust way to split multiple dimensions of grib files at once.